rootfs/usr/lib/bootc/kargs.d/50-playtron.toml

```diff
@@ -0,0 +1,2 @@
+match-architectures = ["x86_64"]
+kargs = ["preempt=full"]
```

Comment:
I've heard from at least one user of winesapOS that this workaround for NVIDIA audio stuttering is no longer needed as of Linux 6.12 thanks to the PREEMPT_RT patches that were merged. Apparently, this audio issue wasn't even NVIDIA specific from my reading. It affected a range of various audio devices.